doc/md/guides/mysql/cd-101.md

```diff
@@ -0,0 +1,86 @@
+---
+id: mysql-cd-101
+title: Detecting drops of foreign key constraints on MySQL
+slug: /guides/mysql/checks/CD101
+tags: [linting, ci, destructive-change, mysql, drop-foreign-key]
+---
+
+## Introduction
+
+This document describes in detail the detection of foreign keys drops for MySQL databases, 
+using Atlas's linting capabilities.
+
+A foreign key constraint can be dropped from a table using the following command:
+```sql
+ALTER TABLE pets DROP FOREIGN KEY owner_id;
+```
+
+Even though this action is not causing data loss and there might be good reasons to perform it on a production database, it can have negative and sometimes unpredictable side effects (some of them are explored below) which should be taken into consideration.
```

Comment:
```suggestion
Even though this action does not cause data loss and there may be good reasons to perform it on a production database, it can have negative and sometimes unpredictable side effects which should be taken into consideration.
```